references/cicd-sharedflow-pipeline/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "apigee-sharedflow-ci-cd",
+	"version": "1.0.0",
+	"description": "Sample Sharedflow for CI/CD Reference",
+	"author": "ssvaidyanathan",
+	"license": "Apache-2.0",
+	"licenses": [
+		{
+			"type": "Apache-2.0",
+			"url": "http://www.apache.org/licenses/LICENSE-2.0"
+		}
+	],
+	"scripts": {
+		"eslint": "eslint -c ./.eslintrc-jsc.yml --format json",
+		"apigeelint": "apigeelint -s ./sharedflowbundle -f table.js",
+		"lint": "npm run eslint && npm run apigeelint"
+	},
+	"devDependencies": {
+		"prettyjson": ">=1.1.3",
+		"apickli": "^2.3.3",
+		"eslint": "^7.0.0",
+		"eslint-config-google": "^0.14.0",
+		"eslint-config-prettier": "^6.11.0",
+		"cucumber": "^7.0.0-rc.0",
+		"jsonfile": ">=2.3.1",
+		"lodash": ">=4.15.0",
+		"bluebird": "^3.3.5",
+		"chai": "^3",
+		"chai-fuzzy": "^1",
+		"mocha": "^2",
```

Comment:
I cant see any unit testing, so we dont need chai, mocha, sinon. 
there isnt a code coverage check so instanbul is not required.

if we want to include these checks, please also add some unit tests